data/campaigns/Heir_To_The_Throne/overworld/00c_northlands.cfg

```diff
@@ -0,0 +1,1573 @@
+        )} {ELSE(
+            [message]
+                speaker=Konrad
+                message=_"The northlands are a dangerous place to be wandering alone, especially in the center of such a fierce snowstorm. We should speak to this woman and see if she needs any help."
+            [/message]
+        )} {/IF}
+        # determine where Konrad spawns on the scenario map
+        {IF} {VARIABLE_CONDITIONAL previous_x greater_than_equal_to 32}
+            {THEN({VARIABLE s23_from e})}
+            {ELSE({VARIABLE s23_from w})}
+        {/IF}
+    )
+
+    #############################
+    # COMPLETED
+    #############################
+    (COMPLETED_PRESTART=
+        {CLEAR_VARIABLE s23_from}
+    )
+    (JUSTCOMPLETED_START=
+
+    )
+    (COMPLETED_MOVETO=
+
+    )
+}
+
+#######################################################################################################################################################
+#                                                                S24 - AN OLD FRIEND
+#######################################################################################################################################################
+#############################
+# GATEKEEPERS
+#############################
+[event]
+    name=prestart
+    {SET_LABEL 30 21 _"Wesmere Forest"}
+    {IF} {VARIABLE_CONDITIONAL bm_wesmere_unlocked not_equals yes} {THEN(
+        {GENERIC_UNIT 2 (Elvish Fighter) 31 19} {FACING nw} {ROLE wesmere_guard}
+        {GENERIC_UNIT 2 (Elvish Ranger)  27 24} {FACING nw} {ROLE wesmere_guard}
+        {GENERIC_UNIT 2 (Elvish Fighter) 37 21} {FACING ne} {ROLE wesmere_guard}
+    )} {ELSE(
+        {GENERIC_UNIT 2 (Elvish Fighter) 28 24} {FACING nw}
+        {GENERIC_UNIT 2 (Elvish Ranger)  28 21} {FACING nw}
+        {GENERIC_UNIT 2 (Elvish Fighter) 37 23} {FACING ne}
+    )} {/IF}
+
+    [capture_village]   
+        side=2
+        x,y=31,22
+        radius=4
+    [/capture_village]
+[/event]
+[event]
+    name=enter hex
+    first_time_only=no
+    {FILTER( id=Konrad {FILTER_ADJACENT role=wesmere_guard} )}
+    {FILTER_CONDITION({VARIABLE_CONDITIONAL bm_wesmere_unlocked not_equals yes})}
+
+    [cancel_action]
+    [/cancel_action]
+    #------------------------
+    # INTRODUCTIONS
+    #------------------------
+    {IF} {VARIABLE_CONDITIONAL bm_wesmere_unlocked not_equals visited} {THEN(
+        [message]
+            role=wesmere_guard {FILTER_ADJACENT id=Konrad}
+            message=_"By order of the lady Chantal, Wesmere forest is closed to outsiders. Be off with you."
+        [/message]
+        [message]
+            speaker=Konrad
+            message=_"Good elf, I am the prince Konrad, child of the Aethenwood and ally of Wesmere. I am no outsider, but come as a friend!"
+        [/message]
+        [message]
+            role=wesmere_guard {FILTER_ADJACENT id=Konrad}
+            message=_"Apologies, but I have my orders. None can enter here. Even you, prince Konrad."
```

Comment:
Since the guard is addressing him by his title, I think this should be Prince Konrad - same for line 513.

(Not sure about Lady Chantal since you added 'the' before it. Without the 'the', I would also capitalise Lady.)